### ISSUE TITLE
[Color Schemes] Visibility of current selection

### ISSUE THREAD

**Issue by jrappen** [Briles/gruvbox#19]
![gruvbox (Dark) (Soft) NDC](https://cloud.githubusercontent.com/assets/8577450/17644829/6d5e3302-6192-11e6-875d-f97173eeac22.png)

Would it be possible to increase the visibility of the current selection? Compare above `gruvbox (Dark) (Soft) NDC` where the selection is only the word `exercitation` in line 3.

**Comment by jrappen** [comment]
`src/scheme.js`

```diff
@@ -17,8 +17,8 @@
           foreground: c.ruler,
           invisibles: c.guide,
           lineHighlight: c.bnp.bg1,
-          selection: c.bnp.bg1,
-          inactiveSelection: c.bnp.bg1,
+          selection: c.bnp.bg3,
+          inactiveSelection: c.bnp.bg3,
           guide: c.guide,
           activeGuide: c.ruler,
           stackGuide: c.guideAlt,
```

**Comment by jrappen** [comment]
It's better with the light themes. I'll have to review the effects...

**Comment by Briles** [comment]
I'm going to do some testing on different color combinations when I have some free time. If we do go forward with any changes, we'll add the changes as a new scheme variant.

**Comment by jrappen** [comment]
This issue only effects the `soft` variants, going to switch to `medium` or `hard`. Closing issue now.

**Comment by nikcorg** [comment]
@Briles @jrappen I think this is still a valid issue, please consider reopening. (I used [Package Resource Viewer](https://github.com/skuroda/PackageResourceViewer) to make the selection more perceptible. Reduced colour opacity, added outline.)

**Comment by jrappen** [comment]
@nikcorg what were your suggested changes?

**Comment by nikcorg** [comment]
@jrappen Can themes have configurable things? If so, maybe make selection outline an option? Also, ensure a negligible amount of contrast between selection colour and background colour. Current default colours on any of the Dark theme variants make selection quite indistinguishable, which is frustrating.

**Comment by jrappen** [comment]
a.) yes
b.) outlines are usually used by linters, not sure that's a good idea.
c.) contrast differs between `soft`, `medium` and `hard` as I said above, did you try all three?

What did you change on your setup with the PackageResourceViewer to get rid of this issue? Reducing the color opacity for the selection doesn't seem to make sense to me.

**Comment by nikcorg** [comment]
I did try all of the variations. The selection colour is very subtle by default. Nigh on invisible, esp. in daylight. 

I have at least (it was a while ago, don't remember exactly):

- changed the `selection` colour (`#69615d`)
- changed the `selectionBorder` colour (`#e8bc0e`)

You're right about opacity changes being ridiculous, not sure what I was thinking.

The selection colour is a hard one, the one I've chose isn't ideal either, but it's "good enough" and achieves the obviousness criteria for me. Apart from that one I really like the colours in this theme. Very calm and subdued.

**Comment by easternwawoman** [comment]
Thanks for this discussion.  I came here, after going through dark themes, trying to find one where highlighted text appears, well, clearly highlighted.  I just installed this and while dark is better, I'm still in need of a few degrees of oomph.  It's like I want to turn the stove up by just one step with the hope it's the subtle increase in contrast that my eyes are seeking.  Boy, dealing with humans is always such a chore.  Thank you for your work.  :-) This theme is nicely done.
